kiosk/sources/rules/owned_kiosk_rule.move

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module kiosk::owned_kiosk_rule {
+    use sui::kiosk::{Self, Kiosk};
+    use sui::transfer_policy::{
+        Self as policy,
+        TransferPolicy,
+        TransferPolicyCap,
+        TransferRequest
+    };
+
+    use kiosk::owned_kiosk;
+
+    /// An item hasn't been placed into the Kiosk before the call.
+    const EItemNotInKiosk: u64 = 0;
+    /// The Kiosk is not owned; the OwnerMarker is not present.
+    const EKioskNotOwned: u64 = 1;
+
+    /// The Rule checking that the Kiosk is an owned one.
+    struct Rule has drop {}
+
+    /// Add the "owned" rule to the KioskOwnerCap.
+    public fun add<T>(policy: &mut TransferPolicy<T>, cap: &TransferPolicyCap<T>) {
+        policy::add_rule(Rule {}, policy, cap, true)
```

Comment:
Nit: bind `true` to a descriptive variable name to make this more readable.

Reply:
It's the shortest placeholder for a dynamic field. I can replace it with `Config {}` to follow the Rules implementation guide but basically no one should ever read this field.

Reply:
I meant something like

```
let name_describing_what_true_means = true;
policy::add_rule(Rule {}, policy, cap, name_describing_what_true_means)
```